.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.